package.json

```diff
@@ -90,9 +89,9 @@
     "flow-bin": "~0.170.0",
     "jest": "^29.7.0",
     "prettier": "^2.8.8",
-    "react": "18.2.0",
+    "react": "18.3.1",
     "react-native": "0.74.1",
-    "react-test-renderer": "18.2.0",
+    "react-test-renderer": "18.3.1",
```

Comment:
It seems the entire `react-test-renderer` package has now been discontinued. See https://github.com/facebook/react/tree/main/packages/react-test-renderer